File: solutions/easy/1539.Kth_Missing_Positive_Number/1539.Kth_Missing_Positive_Number.ts
function findKthPositive(arr: number[], k: number): number {
  let index = 0;
  
  while (index < arr.length) {
    if (arr[index] - index > k) {
      return index + k;
    }
    index++;
  }
  
  return arr.length + k;
}

File: solutions/easy/1539.Kth_Missing_Positive_Number/1539.Kth_Missing_Positive_Number.py
from ast import List


class Solution:
    def findKthPositive(self, arr: List[int], k: int) -> int:
        index = 0

        while index < len(arr):
            if arr[index] - index > k:
                return index + k
            index += 1

        return len(arr) + k
